src/Frontend/actions/menuActions.js

```diff
@@ -12,12 +12,11 @@
 
 import { TRANSFORM_ITEMS } from '../constants/Menu';
 
-export const buildMenuItems = items => ({
-  type:      TRANSFORM_ITEMS,
+export const buildMenuItems = items => (dispatch, state) => dispatch({
+  type:     TRANSFORM_ITEMS,
   items,
-  // TODO add real values
-  auth_data: {
-    is_logged_in: false,
-    is_admin:     false
+  authData: {
+    logged_in: state().user.security.authenticated,
+    is_admin:  false // TODO implement better role handling here
```

Comment:
well, I just added `TODO` comments were it doesn't make sense to create new tasks (see the `userReducer` as reference). I added them because I see in my IDE what needs to be done and this is something that could be done during the rewrite of the login/logout system since it is the second step of the security refactoring which needs to be done after the implementation of the registration /activation refactoring, but basically you're right, we shouldn't mess everything with TODO comments :smile: